### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by gkjohnson**
cc @camargo 

Update the docs with up to date information.

I was looking into running tests, as well, but it seems that changing the CSpice.js function to use modules is causing issues when running in node and babel crashes when trying to transpile it with jest. @camargo is it possible to run node such that it can load both modules and require? That seems to be related to the issue.

Related to #3 

**TODO**

- [x] Fill out remaining TODOs in the README

**Possible Future PR**

- [ ] Remove kernel functions expect for "loadKernelFromBuffer" and "unloadKernel". Change `loadKernelFromBuffer( buffer )` to `loadKernel( buffer, pathKey )` and return the path the kernel is loaded to.
- [ ] See if we can remove the kernel from the file system after furnishing (looks like this works)
- [x] Properly release all pointers in chronos function
- [ ]  Understand why chronos function seems to append extra ascii characters to the result.

### CODE REVIEW

**Comment by camargo** [comment]
I see the `package.json` has been set to ` "type": "module"`, which tells Node (15.1.0) to treat all the `.js` files as ES Modules. See this doc: https://nodejs.org/api/packages.html#packages_determining_module_system
Can we use the `.cjs` extension on the `CSpice.js` file? That way Node should treat it as a CommonJS Module.

**Comment by gkjohnson** [comment]
Yeah I switched the package.json to `type: module` so it would work in node -- I'm using node 14.x. If we switch CSpice.js to use commonjs it won't work in a browser though and we'll have to bundle the example for it to work. Jest also just won't work with modules at all without using `babel-jest` to convert them so I suspect it will still crash even with cjs but we may be able to set it to ignore that file to transform...

If we're going to build the example with something like parcel or rollup I'm more included to just build a commonjs version of the package with rollup and point "main" to "cjs/index.js" in the package.json for node. And then we wouldn't have to deal with babel at all for jest. At some point we'll probably want to look into building a new version of cspice.js with a newer version of emscripten which might afford some more features for module support. We can talk more about it Monday.

These docs are more accurate now, though, and we can look into the other items in upcoming PRs. I can remove the "type": "module" line because it's not doing a whole lot for us right now.

File: README.md [hunk 1]
```diff
@@ -8,312 +8,297 @@ United States Government Sponsorship acknowledged. Any commercial use must be ne
...
 
-1.  Clone the Repository
-```shell
-git clone https://github.com/NASA-AMMOS/timecraftjs.git
+```js
+import * as TimeCraft from 'timecraftjs';
+
+// Load the kernels
+const kernelBuffers = await Promise.all( [
+
```
Comment (camargo) [inline README.md:45]: You certainly love your white-space :)

**Review by camargo — APPROVED** [review]
This looks good to me! Thanks for doing this.

File: README.md (hunk 1 above)
Comment (gkjohnson) [reply README.md:45]: Heh habit from three.js now. I do think the whitespace makes it easier to read which is good for docs but if you'd prefer I can remove it.

**Pull request merged**